### PR TITLE
Disarm on configurator exit

### LINKED ISSUE #2174: Motors Dont Stop Spinning on Disconnect
1. Describe your problem;
    When on the motors tab, you can choose to spin the motors using the GUI by manipulating the motor sliders. If the sliders are active and you disconnect from the Betaflight configurator, the motors continue to spin
2. Include ways to reproduce the problem;
    1. Navigate to the motors tab
    2. Check the box that reads "I understand the risks"
    3. Move the motor sliders
    4. Disconnect by hitting the red button on the top right of the screen. The motors continue to spin.
3. Provide as much information as possible about your hardware and software, including:
- what hardware / software you are using;
    HiFion F405 Stack
- the version of all the software used;
    Betaflight 4.1.0
    Betaflight Configurator 10.7.0

### PR DESCRIPTION
**Pull request by haslinghuis**
ℹ️ Disarm motors on application exit

✔️ Fixes #2174

### CODE REVIEW

**Comment by haslinghuis** [comment]
Not sure this is the right approach but it works

File: src/js/main.js [hunk 1]
```diff
@@ -148,14 +148,23 @@ function closeSerial() {
     }
 }
 
-function closeHandler() {
+async function closeHandler() {
```
Comment (mikeller) [inline src/js/main.js:151]: Not sure if `async` / `await` is supported in the Chrome web app, and we haven't decided to ditch support for it yet.

**Comment by mikeller** [comment]
All in all not sure if this is something that needs changing, but happy to discuss. Can you elaborate on why you think that motors should be stopped when closing configurator?

**Comment by haslinghuis** [comment]
Was trying to solve an [non-]issue, when testing motors in another tab, it looked convenient to stop motors while leaving the configurator without having to return to the motors tab to disable the slider. Already trying without async await but it's still a work in progress.

**Comment by mikeller** [comment]
@haslinghuis: Valid point about convenience - and if a user wants to leave configurator with the motors spinning they can always disconnect USB first.

**Comment by haslinghuis** [comment]
No problem. I am used to how things are now and have no problem with it myself... Closing the PR ;) We should focus on bigger issues.

**Comment by ThisIsJustARandomGuy** [comment]
Can we revisit this? The current way the "I understand the risks" checkbox works goes against what I (and every other pilot I've talked to) expect. Other tabs do not keep the setting changes when the user navigates away unless they press save. The safe thing to do would be to disarm the motors unless some other config option is set. Maybe "Motor debug mode" (somewhat like expert mode but more hidden).

**Pull request closed without merging**